internal/protoc/downloader.go

```diff
@@ -64,7 +71,25 @@ func newDownloader(config settings.Config, options ...DownloaderOption) *downloa
 	if downloader.config.Compile.ProtobufVersion == "" {
 		downloader.config.Compile.ProtobufVersion = vars.DefaultProtocVersion
 	}
-	return downloader
+	if downloader.protocBinPath != "" || downloader.protocWktPath != "" {
+		if downloader.protocURL != "" {
+			return nil, fmt.Errorf("cannot use protoc-url in combination with either protoc-bin-path or protoc-wkt-path")
+		}
+		if downloader.protocBinPath == "" || downloader.protocWktPath == "" {
+			return nil, fmt.Errorf("both protoc-bin-path and protoc-wkt-path must be set")
+		}
+		cleanBinPath := filepath.Clean(downloader.protocBinPath)
+		if _, err := os.Stat(cleanBinPath); os.IsNotExist(err) {
+			return nil, err
+		}
+		cleanWktPath := filepath.Clean(downloader.protocWktPath)
```

Comment:
Want to check that this is a directory and contains the directories google, google/protobuf